lib/src/cmaf/ham/AudioTrack.ts

```diff
@@ -1,11 +1,11 @@
-import { Track } from './Track';
-
+import { Track } from './Track.js';
+import { Segment } from './Segment.js';
 export class AudioTrack extends Track{
     sampleRate:number;
     channels:number;
 
-    constructor(id:string, type:string,codec:string,duration:number,language:string,bandwidth:number, sampleRate:number, channels:number) {
-        super(id, type,codec,duration,language,bandwidth);
+    constructor(id:string, type:string,codec:string,duration:number,language:string,bandwidth:number, sampleRate:number, channels:number, segments:Segment[]) {
+        super(id, type,codec,duration,language,bandwidth,segments);
         this.sampleRate = sampleRate;
         this.channels = channels;
     }

```

Comment:
Please run the linter, indentation should be tabs and a newline should be present at the end of the file